src/blog/2017-12-01/babel-support.pug

```diff
@@ -0,0 +1,49 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Stryker now supports mutating and testing Babel projects!
+        The installation/upgrade scenario's in this blog post can be circumvented by using the [Quickstart](https://stryker-mutator.github.io/quickstart.html) if you don't already use Stryker in your project.
+    
+         ![](../../images/blogs/love-babel.png)
+
+        # New JavaScript mutator
+        In order to support mutating more recent JavaScript features as well as experimental features, we had to change the way we mutate JavaScript code. 
+        We had already released a separate plugin for mutating TypeScript code, but now we have also released a separate plugin for mutating JavaScript code. 
+        It is called the [stryker-javascript-mutator](https://www.npmjs.com/package/stryker-javascript-mutator) and it replaces the current 'ES5' mutator that has always come with Stryker. With this release, the 'ES5' has been deprecated and it will be removed in the future.
+
+        If you use Stryker with JavaScript code on your project, we highly suggest upgrading to the new stryker-javascript-mutator. You can do so by installing the package:
```

Comment:
Please add a new-line here. It isn't needed for markdown-it, but it *is* recommended as it is in the commonmark specification. If we would later switch markdown implementations it could give some unexpected results.